virtme_ng_init/src/main.rs

```diff
@@ -193,7 +193,8 @@ const SYSTEM_MOUNTS: &[MountInfo] = &[
 const USER_SCRIPT: &str = "/run/tmp/.virtme-script";
 
 fn check_init_pid() {
```

Comment:
(maybe this check no longer makes sense, maybe we can remove it? Instead, we could check if `id()` != 1 to know if systemd is being used or not)